apps/docs/components/blocks/NavbarBottom.md

```diff
@@ -8,6 +8,17 @@ hideToc: true
 
 {{ $frontmatter.description }}
 
+::: tip You can make the navbar items links
+Each item in the navbar is an [`SfButton`](../components/button) component. This means that you can make each item a link by using the `tag` prop to make it an `a`, <!-- react -->`NextLink`<!-- end react --><!-- vue -->`NuxtLink`<!-- end vue -->, or any other element/component. [Learn more about this usage in the component documentation.](../components/button#link-as-a-button)
```

Comment:
should we add a version also for react prop: by using the as (for react) tag (for vue) prop?